index.html

```diff
@@ -0,0 +1,230 @@
+            <div class="read-more-item">
+              <div class="send-item">
+                <img src="img/1.png" alt="">
+                <h5>SEND IT NOW</h5>
+              </div>
+              <div class="read-item-footer">
+                <span>Lorem ipsum dolor sit amet, 
+                  consectetur adipiscing elit. Suspendisse 
+                  malesuada sodales risus, a adipiscing 
+                  lorem.</span>
+                <a href="#" class="btn-read">Read more</a>
+              </div>
+            </div>
+            <div class="read-more-item">
+              <div class="send-item">
+                <img src="img/2.png" alt="">
+                <h5>VERY NICE TO SEE</h5>
+              </div>
+              <div class="read-item-footer">
+                <span>Lorem ipsum dolor sit amet, 
+                  consectetur adipiscing elit. Suspendisse 
+                  malesuada sodales risus, a adipiscing 
+                  lorem.</span>
+                <a href="#" class="btn-read">Read more</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="article" class="box">
+        <div class="container">
+          <div class="article-wrap">
+            <div class="img-article">
+              <img src="img/article.jpg" alt="">
+            </div>
+            <div class="article-text">
+              <h5>JUST FOR YOU ON THEMEFOREST</h5>
+              <span>Lorem ipsum dolor sit amet, consectetur adipiscing 
+                elit. Donec interdum mollis risus, in mattis erat convallis 
+                egestas.</span>
+              <a href="#" class="btn-article">Read more</a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="sponsor-data" class="box">
+        <div class="container">
+          <div class="sponsor-wrap">
+            <div class="sponsor-title">
+              <img src="img/sponsor-title.png" alt="">
+              <h5>this template was featured on</h5>
+            </div>
+            <div class="img-sponsor">
+              <a href=""><img src="img/techcrunch.png" alt=""></a>
+              <a href=""><img src="img/promail.png" alt=""></a>
+              <a href=""><img src="img/mashable.png" alt=""></a>
+              <a href=""><img src="img/atlaspix.png" alt=""></a>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="footer">
+        <div class="container">
+          <div class="footer-wrap">
+            <div class="text-footer">
+              <p><span>Elenea &copy;</span> copyright 2013 - Unsubscribe here</p>
+            </div>
+            <div class="socials">
+              <a href=""><img src="img/social-g.png" alt=""></a>
+              <a href=""><img src="img/social-f.png" alt=""></a>
+              <a href=""><img src="img/social-t.png" alt=""></a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+</body>
+</html>
```

Comment:
A file should end with a newline. Here's why: https://stackoverflow.com/a/729795